test.py

```diff
@@ -928,6 +928,7 @@ async def run(self, options: argparse.Namespace) -> Test:
 
         test_path = os.path.join(self.suite.options.tmpdir, self.mode)
         async with get_cluster_manager(self.mode + '/' + self.uname, self.suite.clusters, test_path) as manager:
+            self.args.insert(0, "--mode={}".format(self.mode))
```

Comment:
when we run `test.py --mode dev --mode debug` there will be separate `TopologyTest` instances, with different `self.mode` right? (i.e. `self.mode` is always a single mode, not a list of modes)